crates/compiler/module/src/symbol.rs

```diff
@@ -622,10 +622,43 @@ impl ModuleIds {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct LookedupSymbol {
+    pub symbol: Symbol,
+    pub params: Option<Symbol>,
+}
+
+impl LookedupSymbol {
+    pub fn new(symbol: Symbol, params: Option<Symbol>) -> Self {
+        Self { symbol, params }
+    }
+
+    pub fn no_params(symbol: Symbol) -> Self {
+        Self::new(symbol, None)
+    }
+}
```

Comment:
Yeah, I wanted to move all of `ScopeModules` to `can`, but the `define_builtins!` macro in this file implements `ScopeModules::new` with the builtins already in scope. I guess I could instead have it expose some iterable collection of module ids and name so I can create `ScopeModules` from `can`.

Reply:
> Have you considered trying to elide this abstraction entirely, instead looking up the params in a lookaside table as necessary?
That's what I had initially, but I think having lookups return params when applicable makes it harder to forget them

Reply:
I all moved of `ScopeModules` to `can/scope` as suggested. 

I tried removing the lookup structs and looking up the symbols from the `Var` canonicalization. However, we can't get the params symbol from a symbol's module id, because the same module might be imported multiple times under different aliases. I could lookup the params by module name but we would have to do that twice.